### DISCUSSION TITLE
Delete all my imported highlights

### DISCUSSION THREAD

**Discussion by giuseppepagnoni** [mammuth/kindle-clippings#48] [Q&A] [answered]
HI, how do I delete all my imported clippings in order to do a full fresh re-upload?
Thanks!
Giuseppe

**Comment by mammuth** [comment, marked as answer]
Hi, it's currently only possible to delete individual clippings, there is no bulk deletion sadly.

I created https://github.com/mammuth/kindle-clippings/issues/49 so we have that idea in the backlog.

![OnPaste 20230103-100717](https://user-images.githubusercontent.com/3121306/210327454-a794464b-da4c-485d-80cb-60274328374d.png)
